app.rb

```diff
@@ -1,9 +1,19 @@
 require 'sinatra/base'
+require './lib/peep'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
 
+  get '/messageboard' do
```

Comment:
The naming could be better here, better naming of your route would be **get /peeps**